src/GNNGraphs/gnnheterograph.jl

```diff
@@ -130,26 +130,58 @@ function GNNHeteroGraph(data::EDict;
         ntypes, etypes)
 end
 
+function show_sorted_Dict(io::IO, d::Dict,compact::Bool)
```

Comment:
```suggestion
function show_sorted_Dict(io::IO, d::Dict, compact::Bool)
```
and similar syntax changes in all the added code